packages/flutter/lib/src/material/bottom_app_bar_theme.dart

```diff
@@ -35,8 +35,7 @@ class BottomAppBarTheme extends Diagnosticable {
 
   /// Default value for [BottomAppBar.color].
   ///
-  /// If null, [ThemeData.bottomAppBarColor] is used, if that's null, defaults
-  /// to [Colors.white].
+  /// If null, [ThemeData.bottomAppBarColor] is used.
```

Comment:
This isn't necessary, but it is helpful. Since we're talking about [BottomAppBar] then: 
If null, [BottomAppBar] uses [ThemeData.bottomAppBarColor].

Reply:
Sounds good.